integration-tests/package.json

```diff
@@ -35,6 +35,7 @@
     "docker-compose": "^0.23.8",
     "envalid": "^7.1.0",
     "eslint": "^7.27.0",
+    "eslint-plugin-prettier": "^3.4.0",
```

Comment:
Are there are obsoleted references to `prettier` we could remove?

Reply:
I don't think so — we still use prettier it's just used within `eslint` as a plugin now.

Reply:
I don't think so — we still use `prettier` inside `eslint` as a plugin